src/model/OrganizationInvitation.ts

```diff
@@ -42,16 +43,27 @@ export default class OrganizationInvitation extends Ressource {
   static get(organizationId: string, id: string) {
     const { api_url } = getConfig();
 
-    return super._get<OrganizationInvitation>(`${api_url}${_url}`, { id, organizationId });
+    return super._get<OrganizationInvitation>(`${api_url}${_url}`, {
+      id,
+      organizationId,
+    });
   }
 
-  static query(organizationId: string) {
+  static query(organizationId: string, filterUrl?: string) {
     const { api_url } = getConfig();
 
-    return super._query<OrganizationInvitation>(`${api_url}${_queryUrl}`, { organizationId }, {}, {});
+    return super._query<OrganizationInvitation>(
+      filterUrl
+        ? `${api_url}${_queryUrl}${filterUrl}`
+        : `${api_url}${_queryUrl}`,
+      { organizationId },
+      {},
+      {}
+    );
   }
 
   delete() {
-    return super.delete(this._url);
+    const url = this._url.replace(filterUrl, "");
```

Comment:
Not sure that works for 2 reasons:
 1. `this._url` => [line 5](https://github.com/platformsh/platformsh-client-js/pull/215/files#diff-2492b43f511f5d523e2f4639532023e3d3dbe889f372f5ba6cd5d50f27b02b50L5) `${_queryUrl}/:id`; so doesn't contain the `filterUrl`
  2. because `filterUrl` is a parameter, maybe the value can be different than  [line 8](https://github.com/platformsh/platformsh-client-js/pull/215/files#diff-2492b43f511f5d523e2f4639532023e3d3dbe889f372f5ba6cd5d50f27b02b50R8) and in this case the replace doesn't work